index.html

```diff
@@ -3246,6 +3246,55 @@ <h3>Authorization</h3>
 on top of this specification.
         </p>
       </section>
+
+      <section class="informative">
+        <h3>Reserved Extension Points</h3>
+
+        <p>
+This specification reserves a number of <a>properties</a> serving as possible extension points. While
+some implementers signaled interest in these, their inclusion in this specification
+was considered to be premature; these extension points might be more
+formally defined in future versions of this specification. While these extension
+points are reserved in the base context, their usage is undefined by this
+specification and implementers are cautioned that usage of these properties is
+considered experimental.
+        </p>
+
+        <p class="issue" title="Reserved properties under debate">
+The following extension point properties are under consideration for being
+marked as reserved: `presentationSchema`, `credentialRefresh`, `termsOfUse`,
+`evidence`, `render`, and `confirmationMethod`.
+        </p>
+
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>Reserved&nbsp;Property</th>
```

Comment:
This seems like an embedded registry, sorta like what we have for did spec registries.

Why do we need to repeat the registry in the core spec, why can't we just refer to it where it will exist, like we did with did core?

> Some examples contain terms, both property names and values, that are not defined in this specification. These are indicated with a comment (// external (property name|value)). Such terms, when used in a [DID document](https://www.w3.org/TR/did-core/#dfn-did-documents), are expected to be registered in the DID Specification Registries [[DID-SPEC-REGISTRIES](https://www.w3.org/TR/did-core/#bib-did-spec-registries)] with links to both a formal definition and a JSON-LD context.

- https://www.w3.org/TR/did-core/#conformance

Reply:
Seems relevant to our education vertical specific examples as well.

Perhaps with this change, we could make the examples in the spec reflect real world examples.

Or we could include real world examples in an appendix and update the examples in the core spec to be simpler, and more relevant to the sections they are in, instead of education related details.

Reply:
> Seems relevant to our education vertical specific examples as well.

The examples in the spec feel like an orthogonal issue. The purpose of this section, to establish known extension points that a subset of the group feels are important, is not the same as the text you quoted from the DID Spec.

> Perhaps with this change, we could make the examples in the spec reflect real world examples.

No, that's not the purpose of this PR. We should deal w/ the examples in the spec in a separate PR.

> Or we could include real world examples in an appendix and update the examples in the core spec to be simpler, and more relevant to the sections they are in, instead of education related details.

Yes, +1 to moving away from education-related examples throughout the entire spec. The education-related examples dominated in v1.0 because VCs were largely being adopted by the education sector at the time and a fair number of the people in the group were coming from education-related backgrounds. Additionally, education-related examples seemed to be the least controversial at the time.

In any case, +1 to improving the examples throughout the spec, but we should do that in a separate PR with separate concerns.

Reply:
Resolving since this conversation will not result in changes to this PR.